default.vcl

```diff
@@ -151,6 +151,16 @@ backend upp_page_validator {
   .port = "8080";
 }
 
+backend upp_live_event_validator {
+  .host = "upp-live-event-validator";
+  .port = "8080";
+}
+
+backend upp_internal_live_event_validator {
```

Comment:
Let's remove this backend since we won't be deploying `internal` version of the service for now.